src/templates/en/index.html

```diff
@@ -69,11 +70,50 @@ <h2>
         </button>
       </div>
       <div class="intro-year">2019</div>
-      <div class="intro-image">
-        <img class="home-browser" src="/static/images/home-browser.png" />
-        <img class="camada-751" src="/static/images/camada-751.png" />
-        <img class="camada-759" src="/static/images/camada-759.png" />
-        <img class="camada-752" src="/static/images/camada-752.png" />
+      <div class="intro-image-container">
+        <div class="intro-image">
+          <img class="home-browser" src="/static/images/home-browser.png" />
+          <img class="camada-751" src="/static/images/camada-751.png" />
+          <img class="camada-759" src="/static/images/camada-759.png" />
+          <img class="camada-752" src="/static/images/camada-752.png" />
+        </div>
+      </div>
+      <div class="menu">
+        <li>
+          <a href="#">Table of Contents</a>
+        </li>
+        <li>
+          <a href="#">Contributors</a>
+        </li>
+        <li>
+          <a href="#">Methodology</a>
+        </li>
+        <div>
+          <img src="/static/images/ha-home.png" />
```

Comment:
Link to https://httparchive.org/

Also add `alt` attributes for all `img` elements, even if empty for now.
```suggestion
          <a href="https://httparchive.org/">
            <img src="/static/images/ha-home.png" alt="HTTP Archive home" />
          </a>
```

Reply:
Ditto for each of these images in the footer